eurocropsml/dataset/dataset.py

```diff
@@ -159,9 +159,11 @@ def __getitem__(
         # normalization and scaling to (0,1]
         if self.config.normalize:
             if "S1" in np_data_dict:
+                # range before normalization and scaling: [-50.0,1.0]
                 normalized_s1 = np_data_dict["S1"] * NORMALIZING_FACTOR_S1
                 np_data_dict["S1"] = (normalized_s1 + 1) / 2 + EPSILON
```

Comment:
In this case, we would not use the full target range (0,1] for S1, but only (0, 0.51+eps] no? I would expect the scaling to be
```python
normalized_s1 = (np_data_dict["S1"]+50) * NORMALIZING_FACTOR_S1
np_data_dict["S1"] = normalized_s1 * (1-EPSILON) + EPSILON
```
with `NORMALIZING_FACTOR_S1=1/51`

Reply:
Ah yes, you're right!